apps/site/util/getReleaseAnnounceLink.ts

```diff
@@ -0,0 +1,16 @@
+import semVer from 'semver';
+
+import type { NodeRelease } from '@/types';
+
+export const getReleaseAnnounceLink = (release: NodeRelease) => {
+  // No release announcement for versions <= 18
+  if (semVer.satisfies(release.version, '<=18')) {
+    return null;
+  }
+
+  if (release.status === 'Current') {
+    return null;
+  }
```

Comment:
```suggestion
  if (semVer.satisfies(release.version, '<=18') || release.status === 'Current') {
    return null;
  }
```

We can do it in "one" condition